lib/view_component/test_helpers.rb

```diff
@@ -41,14 +41,49 @@ def refute_component_rendered
     # @param component [ViewComponent::Base, ViewComponent::Collection] The instance of the component to be rendered.
     # @return [Nokogiri::HTML]
     def render_inline(component, **args, &block)
-      @rendered_component =
+      @rendered_content =
```

Comment:
Super minor, but I wonder if this is being used directly by any libraries/apps and if renaming it would break anything.

Reply:
I do wonder that too. I'll add a note to the changelog.

Reply:
This indeed is used by lots of people! #1372 and #1373 offer two fixes.

I wonder if we should update the [documentation] to more explicitly direct people towards writing expectations that use `page` if they are using RSpec and Capybara?

```ruby
expect(page).to have_link("Home page", href: root_path)
```

[documentation]: https://viewcomponent.org/guide/testing.html#rspec-configuration

Reply:
Related issue: https://github.com/github/view_component/issues/1375